about.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Home &mdash; Nataliia Getlin's Personal Portfolio Site</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Place this tag in your head or just before your close body tag. -->
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
+
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+        header {
+            background-image: url('./images/header_background.jpg');
+            background-repeat: no-repeat;
+            background-attachment: fixed;
+            background-size: cover;
+        }
+    </style>
+</head>
+
+<body>
+    <main id="home">
+        <header class="page-header">
+            <h1>About Me</h1>
+        </header>
+        <nav>
+            <ul id="main-menu">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="./portfolio.html">Portfolio</a></li>
+                <li><a href="./hobby-blog.html">Blog</a></li>
+            </ul>
+        </nav>
+        <section>
+
+            <p class="about-description">
+                Hi, my name is Natasha.<br>
+                I am originally from Ukraine. <br>I fluently speak Russian, Ukrainian and English.<br>
+                I live in the Seattle area, WA.<br><br>
+                I have a background in copywriting, graphical design, marketing, public relations and fundraising,
+                working both in
+                non-profit and commercial sectors.<br><br>
+                Currently I am a Software Engineer at training @ Ada Developers Academy.<br><br>
+                I have 2 awesome daughters, who support me along my journey to become a Software Developer.<br><br>
+                I love travelling, hiking, cooking, reading.<br><br>
+                I am always looking for efficint ways to learn and grow professionally and personally. <br><br>I am happy to
+                make new
+                connections and share my knowlegde.
+            </p>
```

Comment:
One cool thing we can do to stay away from `br` elements is to put this text into many `p` elements. They have default styling already built into them to break up new lines.